File: C++/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Exercises/Solutions/P02_SummerOutfit.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {

	string strCelsius, dayTime;
	getline(cin, strCelsius);
	int celsius = stoi(strCelsius);
	getline(cin, dayTime);

	string outfit, shoes;
	if (dayTime == "Morning") {
		if (10 <= celsius && celsius <= 18) {
			outfit = "Sweatshirt";
			shoes = "Sneakers";
		} else if (18 < celsius && celsius <= 24) {
			outfit = "Shirt";
			shoes = "Moccasins";
		} else if (25 <= celsius) {
			outfit = "T-Shirt";
			shoes = "Sandals";
		}
	} else if (dayTime == "Afternoon") {
		if (10 <= celsius && celsius <= 18) {
			outfit = "Shirt";
			shoes = "Moccasins";
		}
		else if (18 < celsius && celsius <= 24) {
			outfit = "T-Shirt";
			shoes = "Sandals";
		}
		else if (25 <= celsius) {
			outfit = "Swim Suit";
			shoes = "Barefoot";
		}
	} else if (dayTime == "Evening") {
		if (10 <= celsius && celsius <= 18) {
			outfit = "Shirt";
			shoes = "Moccasins";
		}
		else if (18 < celsius && celsius <= 24) {
			outfit = "Shirt";
			shoes = "Moccasins";
		}
		else if (25 <= celsius) {
			outfit = "Shirt";
			shoes = "Moccasins";
		}
	}

	cout << "It's " << celsius << " degrees, get your " << outfit << " and " << shoes << "." << endl;

	return 0;
}

File: Python/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Exercises/Solutions/P02_SummerOutfit.py
weather = int(input())
time = input()

Outfit = 0
Shoes = 0

if time == "Morning":
    if 10 <= weather <= 18:
        Outfit = "Sweatshirt"
        Shoes = "Sneakers"
    elif 18 < weather <= 24:
        Outfit = "Shirt"
        Shoes = "Moccasins"
    else:
        Outfit = "T-Shirt"
        Shoes = "Sandals"

if time == "Afternoon":
    if 10 <= weather <= 18:
        Outfit = "Shirt"
        Shoes = "Moccasins"
    elif 18 < weather <= 24:
        Outfit = "T-Shirt"
        Shoes = "Sandals"
    else:
        Outfit = "Swim Suit"
        Shoes = "Barefoot"

if time == "Evening":
    Outfit = "Shirt"
    Shoes = "Moccasins"

print(f'It\'s {weather} degrees, get your {Outfit} and {Shoes}.')
